### ISSUE TITLE
Update translation in diferents languages

### ISSUE THREAD

**Issue by xaca** [xaca/coding-music#87]
We update instructions, point 6. We need update the others files.

**Comment by harshalkh** [comment]
I am taking this

**Comment by harshalkh** [comment]
Created PR #89 Please review

**Comment by Tititesouris** [comment]
I'm working on fixing the English and French translations, and I have a question regarding the orginal spanish sentence:
"Dentro de cada carpeta hay un archivo con formato markdown con el nombre del genero, este se debe editar, teniendo en cuenta de no repetir canción y **de agregarla al final del archivo, según el orden que lleve el listado**."
How can it be both at the end of the file AND in the order of the list ?

**Comment by xaca** [comment]
@Tititesouris Yes, you're right, the redaction is incorrect. I'll change spanish file.
